ignite/metrics/ssim.py

```diff
@@ -36,14 +36,17 @@ class SSIM(Metric):
         ``y_pred`` and ``y`` can be un-normalized or normalized image tensors. Depending on that, the user might need
         to adjust ``data_range``. ``y_pred`` and ``y`` should have the same shape.
 
-        .. code-block:: python
-
-            def process_function(engine, batch):
-                # ...
-                return y_pred, y
-            engine = Engine(process_function)
-            metric = SSIM(data_range=1.0)
-            metric.attach(engine, "ssim")
+        >>> def process_function(engine, batch):
+        ...     y_pred, y = batch
```

Comment:
Can't it work with code-block ? I'm not a big fan of `>>>`

Reply:
It does not work AFAIK.

Reply:
With `>>>` when code is copied it is with these useless `>>>`. I think it would be better to use other tags instead of `>>>` which requires each line modification.

Reply:
sphinx-copybutton can be modified to strip `>>>` in `conf.py`.

Reply:
I agree the @vfdev-5's remark about `>>>`. Maybe we could explore an alternative and see ?

Reply:
@vfdev-5 @sdesrozis change to using `.. testcode::` and `.. testoutput::`. If we are using those, we need to use `print` to test the expected output.

Reply:
Does the rendering of the doc show the output ?

Reply:
>  If we are using those, we need to use `print` to test the expected output.
Not sure to understand

Reply:
I mean if we are using `.. testcode::` and `.. testoutput::`, we need to use `print` like `print(state.metrics['ssim'])` to test the expected output.

> Does the rendering of the doc show the output ?

Yes, it shows.